mmdet3d/models/dense_heads/shape_aware_head.py

```diff
@@ -0,0 +1,640 @@
+import numpy as np
+import torch
+from mmcv.cnn import ConvModule, bias_init_with_prob, normal_init
+from torch import nn as nn
+
+from mmdet3d.core import box3d_multiclass_nms, limit_period, xywhr2xyxyr
+from mmdet.core import multi_apply
+from mmdet.models import HEADS
+from ..builder import build_head
+from .anchor3d_head import Anchor3DHead
+
+
+@HEADS.register_module()
+class BaseShapeHead(nn.Module):
```

Comment:
Considering reuse [BaseConvBboxHead](https://github.com/open-mmlab/mmdetection3d/blob/master/mmdet3d/models/dense_heads/base_conv_bbox_head.py#L9).

Reply:
I would prefer to using current structure for consistency of the input names and meanings with the anchor3dhead. Reusing BaseConvBboxHead can not make it obviously simpler or clearer.